commands/docker/docker-registry-login.md

```diff
@@ -0,0 +1,17 @@
+### Login to docker registry
+
+`docker login [-u {{USERNAME}}] [-p {{PASSWORD}}] [SERVER]`
```

Comment:
`docker login [[-u {{USERNAME}}]] [[-p {{PASSWORD}}]] [[SERVER]]`. I feel this compromises the readability. 
So, I used the above one. Let me know which looks better.

Reply:
single brackets looks fine